src/AspNet.Security.OAuth.Validation/Events/IValidationEvents.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Threading.Tasks;
+
+namespace AspNet.Security.OAuth.Validation.Events
+{
+    public interface IValidationEvents
```

Comment:
I thought of that. It was just a fast-code (just as unhealthy as fast-food). I just wanted to know if it's what we need :+1: